src/components/complibrary/buildproductcardfavorites.jsx

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import {CardText,Card,CardTitle, CardMenu, IconButton} from 'react-mdl';
+import {Link} from 'react-router-dom';
+
+class BuildProductCardFavorites extends Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            //empty for now
+        }
+    }
+    
+
+    componentDidMount(){
+    }
+
+    render() {
+        const product = this.props.productFromParent;
+        var productNameEllipsis = product.productName.toString().substring(0,29) + "...";
+        return(
+            <div>
```

Comment:
is this `div` needed? can it be a `fragment`? https://reactjs.org/docs/fragments.html